geopandas/tests/test_op_output_types.py

```diff
@@ -0,0 +1,276 @@
+import pandas as pd
+import pyproj
+import pytest
+
+from shapely.geometry import Point
+
+from geopandas import GeoDataFrame, GeoSeries
+
+crsgs_osgb = pyproj.CRS(27700)
+crs_wgs = pyproj.CRS(27700)
+# TODO need to check that behaviour works with non default geom col name
+
+
+class TestDataFrameMethodReturnTypes:
+    def setup_method(self):

```

Comment:
I know we still use `setup_method` in some existing test classes, but I think in general we prefer using pytest fixtures. So if you would like to rewrite this slightly, that would be nice to do for new tests.

We can have a fixture for the dataframe:

```
@pytest.fixture
def df():
    return GeoDataFrame(...)
```

and then we could maybe also parametrize this fixture with the geometry column name being  "geometry" and something else.